x-pack/platform/test/alerting_api_integration/common/plugins/actions_simulators/server/jira_simulation.ts

```diff
@@ -155,6 +160,45 @@ export function initPlugin(router: IRouter, path: string) {
       });
     }
   );
+
+  router.post(
```

Comment:
Because this is a Jira simulator and the new route is not a real Jira endpoint like the previous ones, how easy is it to use an existing route (for example, the `/rest/api/2/issue`) and validate the headers only if they are defined?